code/server/validations/authValidation.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// Import the module for validation checks
+const { check } = require('express-validator');
+
+// Validation checks for registration of a new user
+exports.signup = [
+    check('firstname').isString().notEmpty(),
+    check('lastname').isString().notEmpty(),
+    check('email').isEmail(),
+    check('mobile').isString().notEmpty(),
+    check('role').isString().notEmpty(),
```

Comment:
Try to check that role is on a whitelist, where the whitelist is = ["Hiker", "Local Guide",  "Hut Worker", "Emergency Operator"]